appveyor.yml

```diff
@@ -74,7 +86,7 @@ test_script:
   - del /Q coverage.xml
 
   - ps: |
-      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache
+      & py.test --junitxml .\integrationtests.xml integrationtests.py --cov=clcache --cov=storage
```

Comment:
I thought If no test are run - it should be clear that storage.py coverage is 0% - so we can track regressions